### PR TITLE
Address the issue of editing the management IP of a device that alrea…

### PR DESCRIPTION
**Pull request by Abhishek-121**
…dy exists in the inventory, add log level as debug in dnac.py get_task_details api.

In this commit - 

-- Address the issue of When editing the management IP of a device that already exists in the inventory, it should return a failure.

### CODE REVIEW

File: plugins/modules/inventory_intent.py [hunk 1]
```diff
@@ -2560,46 +2560,50 @@ def get_diff_merged(self, config):
...
-                    )
-                    self.log("Received API response from 'sync_devices': {0}".format(str(response)), "DEBUG")
-
-                    if response and isinstance(response, dict):
-                        task_id = response.get('response').get('taskId')
+                    if playbook_params['updateMgmtIPaddressList']:
+                        new_mgmt_ipaddress = playbook_params['updateMgmtIPaddressList'][0]['newMgmtIpAddress']
+                        if new_mgmt_ipaddress in self.have['device_in_dnac']:
+                            self.status = "failed"
+                            self.msg = "Device with IP address {0} already exists in Inventory".format(new_mgmt_ipaddress)
```
Comment (madhansansel) [inline plugins/modules/inventory_intent.py:2567]: inventory.

IP address '{0}'

File: plugins/modules/inventory_intent.py [hunk 2]
```diff
@@ -2560,46 +2560,50 @@ def get_diff_merged(self, config):
...
+                            while True:
+                                execution_details = self.get_task_details(task_id)
 
+                                if execution_details.get("endTime"):
+                                    self.status = "success"
+                                    self.result['changed'] = True
+                                    self.result['response'] = execution_details
+                                    self.msg = "Devices {0} present in Cisco Catalyst Center and updated successfully".format(str(device_to_update))
+                                    self.log(self.msg, "INFO")
+                                    self.status = "success"
```
Comment (madhansansel) [inline plugins/modules/inventory_intent.py:2591]: also written in line num 2586

File: plugins/modules/inventory_intent.py (hunk 1 above)
Comment (Abhishek-121) [reply plugins/modules/inventory_intent.py:2567]: added.

File: plugins/modules/inventory_intent.py (hunk 2 above)
Comment (Abhishek-121) [reply plugins/modules/inventory_intent.py:2591]: removed.

**Review by madhansansel — APPROVED** [review]

**Pull request merged**